sample-app/index.js

```diff
@@ -58,7 +58,6 @@ mongoStorage.connect(
         fb.installWebhook(app, config.facebook.webhook_path, config.facebook.webhook_verification_token, botEngine, scheduler)
         tg.installWebhook(app, config.telegram.webhook_host, config.telegram.webhook_path, botEngine, scheduler)
         storage.createIndex()
-        storage.clean()
```

Comment:
Ok, this was just an unfortunate commit grouping. Please, don't leave such things in the history, they make it hard to review